src/utils.js

```diff
@@ -30,11 +36,36 @@ exports.paths = {
 exports.pkg = pkg
 // TODO: get this from aegir package.json
 exports.browserslist = '>1% or node >=10 and not ie 11 and not dead'
-
 exports.repoDirectory = path.dirname(pkgPath)
 exports.fromRoot = (...p) => path.join(exports.repoDirectory, ...p)
 exports.hasFile = (...p) => fs.existsSync(exports.fromRoot(...p))
 exports.fromAegir = (...p) => path.join(__dirname, '..', ...p)
+exports.hasTsconfig = exports.hasFile('tsconfig.json')
+
+exports.parseJson = (contents) => {
```

Comment:
💭 might be worth using TS native config parsing functionality, see here for details
https://github.com/Gozala/typescript-error-reporter-action/blob/4dffc951c01ed334b9f2f9648a01e6e4c7069ed4/src/index.ts#L46